blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -50,13 +50,15 @@ class ResultsList extends Component {
       resultList: {},
       seeMore: true,
       placeholderResizedImageOptions: {},
-      count: 0,
     };
     this.phrases = getTranslatedPhrases(getProperties(props.arcSite).locale || 'en');
-    this.fetchStories(false);
     this.fetchPlaceholder();
   }
 
+  componentDidMount() {
+    this.fetchStories(false);
```

Comment:
Hi @beltrancaliz I would request you review this PR again. I figured out some issue with content caching that causing the overlaps item issue with the last PR. I refactor something to get it works again. Thanks all for your help to review this!!!